### ISSUE TITLE
can we update gene name files

### ISSUE THREAD

**Issue by ValWood** [pombase/website#731]
Last updated October and November
ftp://ftp.pombase.org/pombe/names_and_identifiers/

**Comment by kimrutherford** [comment]
As a stopgap, Midori has updated the gene names file manually.

I'll fix it to update nightly.

**Comment by kimrutherford** [comment]
I've fixed this for now by replacing the allNames.tsv in SVN with a symbolic link to the nightly file.  I'll check tomorrow to make sure this works OK in the nightly load.

**Comment by ValWood** [comment]
Did this work OK, If so I'll update Maria and co.

**Comment by kimrutherford** [comment]
I think that's a different file.  Sorry about the bad choice of file names.  We have:

  - all_ids.tsv - created for Maria and contains the product, chromosome ID and feature type

and

   - allNames.tsv - which contains just the gene ID, name and synonyms

**Comment by mah11** [comment]
Should we rename the files?

allNames.tsv -> gene_IDs_names.tsv

all_ids.tsv -> gene_IDs_names_products.tsv

or similarly informative wordings (would also move away from the mildly annoying mix of underscores and CamelCase ...)

**Comment by kimrutherford** [comment]
I'm in favour of that

**Comment by ValWood** [comment]
Yes that sounds good.
We would need to let people know.
pombelist, EsyN, danny Bitton, EBI (Maria, HGNC) anyone else?

**Comment by ValWood** [comment]
Are we going to rename the files?
Maria asked me to let her know when the new file is in place?

**Comment by ValWood** [comment]
We need to decide what to do here so we can let Maria know  where the file is located.

**Comment by ValWood** [comment]
It doesn't appear on the live ftp at the moment.

**Comment by mah11** [comment]
I'm happy to rename the files, and I can update static pages to document the new names, and which old names go with which new ones.

**Comment by ValWood** [comment]
OK when that's done I'll let people know....

**Comment by kimrutherford** [comment]
I've made the code change but won't come through until Thursday morning.  As Midori is away I'll update the static pages.  I'll do that once I've checked things on Thursday.

**Comment by kimrutherford** [comment]
> I've made the code change but won't come through until Thursday morning.

I updated everything after the main load finished.

**Comment by kimrutherford** [comment]
Whoops, re-opening.

> pombelist, EsyN, danny Bitton, EBI (Maria, HGNC) anyone else?

We need to update these people.

There are still copies of the files with the old names on the FTP.  I thought I'd leave them in place for now while we let people know.

**Comment by ValWood** [comment]
files are empty
ftp://ftp.pombase.org/pombe/names_and_identifiers/

Val from JFK...Grrrrrr flights cancelled.

**Comment by kimrutherford** [comment]
> files are empty

They are symbolic links so they look empty but they should work if you click on them.

> Val from JFK...Grrrrrr flights cancelled.

Drat!

**Comment by ValWood** [comment]
> They are symbolic links so they look empty but they should work if you click on them.

so won't the files have these names?

**Comment by ValWood** [comment]
Also should we remove the old  file  (or make that a symbolic link to the new file)? in case people keep using it?

**Comment by kimrutherford** [comment]
> so won't the files have these names?

I don't follow.  The files are re-created nightly and copied to here: ftp://ftp.pombase.org/nightly_update/misc/

We made symbolic links some weeks ago so that the files would appear to be in more sensible directory - pombe/names_and_identifiers/.  Having links is a short term fix.  We'll do something better in future.

> Also should we remove the old file

I was planning to remove the old files once everyone using them have been told in case they are downloading automatically using the old names.

> I wan't to announce but can you summarize the changes for me?

This file: ftp://ftp.pombase.org/pombe/names_and_identifiers/allNames.tsv
is now: ftp://ftp.pombase.org/pombe/names_and_identifiers/gene_IDs_names.tsv

and we've added a new file including products in the names_and_identifiers directory:
  ftp://ftp.pombase.org/pombe/names_and_identifiers/gene_IDs_names_products.tsv
(I think this is the one HGNC need)

Anyone getting files from the nightly_update directory should be encouraged to use the names_and_identifiers directory instead.

**Comment by ValWood** [comment]
ftp://ftp.pombase.org/pombe/names_and_identifiers/README
needs updating too

**Comment by ValWood** [comment]
Did the HGNC file name change too? or is that the same.
I told all the other people.

**Comment by kimrutherford** [comment]
> Did the HGNC file name change too? or is that the same.

They were using:
  ftp://ftp.pombase.org/nightly_update/misc/all_ids.tsv
but now should use:
  ftp://ftp.pombase.org/pombe/names_and_identifiers/gene_IDs_names_products.tsv
which is the one we're going to keep and maintain in the future.

**Comment by ValWood** [comment]
This is a duplicate of 
https://github.com/pombase/website/issues/894

**Comment by ValWood** [comment]
closing in favour of https://github.com/pombase/website/issues/894
which is shorter
